routes/users.js

```diff
@@ -40,5 +40,12 @@ router.patch("/profileURL", authenticate, userValidator.updateProfileURL, users.
 router.patch("/rejectDiff", authenticate, authorizeRoles([SUPERUSER]), users.rejectProfileDiff);
 router.patch("/:userId", authenticate, authorizeRoles([SUPERUSER]), users.updateUser);
 router.get("/suggestedUsers/:skillId", authenticate, authorizeRoles([SUPERUSER]), users.getSuggestedUsers);
+router.patch(
+  "/servername/:userId",
+  authenticate,
+  authorizeRoles([SUPERUSER]),
+  checkIsVerifiedDiscord,
```

Comment:
Please update the JSDoc comment on this middleware. As it's mentioned that this middleware is not needed to be called explicitly

Check line number 5 below

https://github.com/Real-Dev-Squad/website-backend/blob/8e951ada9106cada0020d08460241f0652bd0e81/middlewares/verifydiscord.js#L1-L15